requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas
+sklearn
+scikit-image
```

Comment:
Where are `pandas` and `scikit-image` used? Maybe we can leave comments for certain items in this root level `requirements.txt` since it can be difficult to know which is used where.

Reply:
Pandas used in `max_trials_callback.py`:
```
Traceback (most recent call last):
|   File "max_trials_callback.py", line 35, in <module>
|     trials = study.trials_dataframe()
|   File "/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/optuna/study.py", line 782, in trials_dataframe
|     return _trials_dataframe(self, attrs, multi_index)
|   File "/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/optuna/_dataframe.py", line 26, in _trials_dataframe
|     _imports.check()
|   File "/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/optuna/_imports.py", line 86, in check
|     raise ImportError(message) from exc_value
| ImportError: Tried to import 'pandas' but failed. Please make sure that the package is installed correctly to use this feature. Actual error: No module named 'pandas'.
```

sklearn used in `enqueue_trial.py`:
```
| Traceback (most recent call last):
|   File "enqueue_trial.py", line 9, in <module>
|     import sklearn.datasets
| ModuleNotFoundError: No module named 'sklearn.datasets'
| Traceback (most recent call last):
|   File "enqueue_trial.py", line 9, in <module>
|     import sklearn.datasets
| ModuleNotFoundError: No module named 'sklearn.datasets'
```

Will note accordingly.

Reply:
Thanks!